custom_mutators/aflpp_baseline/aflpp.c

```diff
@@ -0,0 +1,100 @@
+// This simple example just creates random buffer <= 100 filled with 'A'
+#include "afl-fuzz.h"
+
+#include <stdint.h>
+#include <stdlib.h>
+#include <string.h>
+#include <stdio.h>
+#include <sys/types.h>
+#include <sys/ipc.h>
+#include <sys/msg.h>
+#include <time.h>
+
+#ifndef _FIXED_CHAR
+  #define _FIXED_CHAR 0x41
+#endif
+
+typedef struct my_mutator {
+
+  afl_state_t *afl;
+
+  // Reused buffers:
+  u8 *fuzz_buf;
+
+} my_mutator_t;
+
+int hexCharToVal(char c) {
```

Comment:
Is  this function useful?

Reply:
Yes, it writes hex string to the seed buffer.